src/Appwrite/Platform/Tasks/ScheduleBase.php

```diff
@@ -60,25 +60,39 @@ public function action(Group $pools, Database $dbForConsole, callable $getProjec
          * @var Document $schedule
          */
         $getSchedule = function (Document $schedule) use ($dbForConsole, $getProjectDB): array {
+            $function = null; /** for ScheduleExecutions */
+
             $project = $dbForConsole->getDocument('projects', $schedule->getAttribute('projectId'));
+            $projectDB = $getProjectDB($project);
 
             $collectionId = match ($schedule->getAttribute('resourceType')) {
                 'function' => 'functions',
                 'message' => 'messages',
                 'execution' => 'executions'
             };
 
-            $resource = $getProjectDB($project)->getDocument(
+            $resource = $projectDB->getDocument(
```

Comment:
```suggestion
            $resource = $dbForProject->getDocument(
```
we conventionally call it `dbForProject`